.github/workflows/ml-pipelines-ci.yml

```
@@ -43,7 +43,10 @@ jobs:
           python-version: "3.14"
 
       - name: Install DVC
-        run: pip install 'dvc[s3]'
+        # Pin the resolved version and install wheels only, so no source-package
+        # setup scripts run during dependency installation. Renovate keeps the
+        # pin current.
+        run: pip install --only-binary :all: 'dvc[s3]==3.67.1'
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)